.eslintrc.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
     'arrow-body-style': ['warn', 'as-needed'],
     'no-param-reassign': ['error', { props: false }],
     'import/prefer-default-export': 'off',
+    'import/no-extraneous-dependencies': 'off',
+    'import/no-unresolved': 'off',
```

Comment:
Why we need to add this?

Reply:
I've removed these since `@hospitalrun/components` is now in the package.json